WikfixResume/form.html

```diff
@@ -30,6 +36,9 @@
                             <span class="sr-only">(current)</span>
                         </a>
                     </li>
+                    <li class="nav-item active">
+                        <a class="nav-link" href="#" onclick="logoutAlert();">Log out</a>
```

Comment:
Let's redirect to login page on logout click! Alerts are annoying. Also, localStorage should be cleared on logout click